owncloudApp/src/main/java/com/owncloud/android/presentation/releasenotes/ReleaseNotesViewModel.kt

```diff
@@ -110,6 +110,11 @@ class ReleaseNotesViewModel(
                 subtitle = R.string.release_notes_4_3_0_subtitle_display_name_truncation,
                 type = ReleaseNoteType.BUGFIX
             ),
+            ReleaseNote(
+                title = R.string.release_notes_4_3_0_title_accessibility_improvements,
+                subtitle = R.string.release_notes_4_3_0_subtitle_accessibility_improvements,
+                type = ReleaseNoteType.ENHANCEMENT
```

Comment:
`CHANGE` and `ENHANCEMENT` does not have to do with technical or non-technical feature. I understand the meaning like:

change -> something different (a change of behaviour of any feature, a UI item placed in different place...)
enhancement -> something best (doing the same but improved or more powerful)

it's no easy to choose one of them for this feature. Since we are adding more accessibility features that did not exist before, `ENHANCEMENT` fits, but also changes over existing items are done too, making `CHANGE` valid.

So, both are valid but IMO `ENHANCEMENT` sounds like more "we give you more than you had". I'd vote for this.

Reply:
Yes, that's what I meant... In this case there have been technical changes, they don't need to be technical to place a `CHANGE` tag, but in this case, they are. I also agree choosing `ENHANCEMENT` for this case 👍